docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference and the success of the conference relies on our volunteers. The conference organizers are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (clocking in at an average of three hours each), you will receive free registration for the conference.
+
+We are looking for folks to help out in these roles:
+
+- Ballroom setup - Prepping the main conference hall for the main conference, including laying out cables/power strips, arranging tables/chairs, and other physical labor
+- Writing day - Help with registering people and general information inquiries, most likely directions to various rooms and schedule questions, assist with other duties as assigned
+- Registration/Info - Registering people in the main conference hall and answering any questions
+- Speakers - Finding speakers and making sure that they're where they need to be when it comes time to speak
+- Talk Timing - Devising a system for timing talks and warning speakers when they're running out of time
```

Comment:
Don't we alredy have a timing system?

Reply:
The timers have cards, though (if I remember correctly) last year there was a timing computer.